hoot-core/src/main/cpp/hoot/core/conflate/splitter/DualWaySplitter.cpp

```diff
@@ -288,6 +291,10 @@ boost::shared_ptr<OsmMap> DualWaySplitter::splitAll()
     cout << endl;
   }
 
+  //  Remove the un-needed nodes from the original way that aren't part of any other way now
+  for (unordered_set<long>::iterator it = _nodes.begin(); it != _nodes.end(); ++it)
+    RemoveNodeOp::removeNode(_result, *it, true);
+
```

Comment:
Curious thing about the splitter is that after the two new ways were formed and the old way removed, it left all of the nodes from the original way there in the file as orphaned nodes.  This goes through and removes them after all of the ways in the map are split.  See the explanation of the `true` flag that I added to the remove op below.